Corona-Warn-App/src/main/res/values-de/release_info_strings.xml

```diff
@@ -17,26 +17,22 @@
 
     <!-- XHED: Titles for the release info screen bullet points -->
     <string-array name="new_release_title">
-        <item>Aufzeichnung von Fehleranalyse-Protokollen</item>
-        <item>Anlegen von Schnelltest-Profilen</item>
+        <item>Digitaler Impfnachweis</item>
     </string-array>
 
     <!-- XTXT: Text bodies for the release info screen bullet points -->
     <string-array name="new_release_body">
-        <item>Sie können nun im Rahmen des technischen Supports für die App ein Fehlerprotokoll erstellen, das die Schritte aufzeichnet, die Sie in der App ausführen. Somit wird die Analyse von möglichen Fehlern erleichtert, welches zu einer schnelleren Fehlerbehebung führen kann.</item>
-        <item>Sie können nun ein Schnelltest-Profil mit Ihren persönlichen Daten anlegen. Ihr Profil kann dann an Teststellen per QR-Code schnell und einfach gescannt werden.</item>
+        <item>Sie können jetzt Ihre Impfungen in der App hinzufügen und jederzeit per QR-Code als Impfzertifikat vorweisen. 14 Tage nachdem Sie alle notwendigen Impfungen erhalten haben, können Sie den vollständigen Impfnachweis in der App vorzeigen.</item>
```

Comment:
Please change text to "Sie können jetzt Ihre Impfzertifikate in der App hinzufügen und per QR-Code vorweisen. 14 Tage nach der letzten Impfung zeigt die App den vollständigen Impfschutz an."